core/ipc/sink/kafka/server/src/main/java/org/opennms/core/ipc/sink/kafka/server/offset/KafkaOffsetProvider.java

```diff
@@ -217,7 +220,7 @@ public void shutdown() {
     }
 
     public KafkaOffsetProvider() {
-        this(new OnmsKafkaConfigProvider(KafkaSinkConstants.KAFKA_CONFIG_SYS_PROP_PREFIX), null);
+        this(new OnmsKafkaConfigProvider(KafkaSinkConstants.KAFKA_CONFIG_SYS_PROP_PREFIX, KAFKA_COMMON_CONFIG_SYS_PROP_PREFIX), null);
```

Comment:
For consistency wouldn't be better to use KAFKA_CONFIG_SYS_PROP_PREFIX instead of KafkaSinkConstants.KAFKA_CONFIG_SYS_PROP_PREFIX with the import static org.opennms.core.ipc.common.kafka.KafkaSinkConstants.KAFKA_CONFIG_SYS_PROP_PREFIX ?